src/components/AddUser/AddUser.tsx

```diff
@@ -16,48 +17,56 @@ export const AddUser = () => {
 
   const [rows, setRows] = useState<Row[]>([]);
 
-  useEffect(() => {
-    const { getInvitations } = users.thunk;
-    (async () => {
-      const result = await dispatch(getInvitations({ id: id as string }));
+  const getInvitationsHandler = async () => {
+    try {
```

Comment:
async thunk already has try catch inside

Reply:
let's keep it, if something crash in map